modules/tools/pipeline/providers/reconciler/taskrun/taskop/wait.go

```diff
@@ -109,6 +110,12 @@ func (w *wait) WhenDone(data interface{}) error {
 			}
 		}
 	}
+	if statusDesc.Desc != "" {
```

Comment:
I think you will treat some success desc as error?

Reply:
In fact desc field only contains error messages when taskOP reconcile, success desc it's not necessary during task reconcile